OpenRA.Game/Server/Server.cs

```diff
@@ -1163,6 +1163,12 @@ public Session.Client GetClient(Connection conn)
 			return LobbyInfo.ClientWithIndex(conn.PlayerIndex);
 		}
 
+		/// <Remarks> Does not check if client is admin </Remarks>
+		public bool CanKickClient(Session.Client client)
+		{
+			return State != ServerState.GameStarted || client.IsObserver || worldPlayers.FirstOrDefault(p => p?.ClientIndex == client.Index)?.Outcome == WinState.Lost;
```

Comment:
Shouldn't this be `!= WinState.Undefined`?

Reply:
It could if we want to allow you to kick all players after the game has ended.

Reply:
Changed it to undefined